robottelo/ui/hosts.py

```diff
@@ -61,6 +61,8 @@ def _configure_hosts(self, arch=None, custom_ptable=None, cv=None,
             self.wait_until_element(locators['host.ip']).send_keys(ip_addr)
         # Operating system tab
         if arch or os or media or ptable or custom_ptable or root_pwd:
+            self.scroll_page()
+            self.wait_for_ajax()
```

Comment:
Ok.. I retested and realized we required `wait_for_ajax()`. 

Reason: when selenium selects the required dropdown options from new host form, then UI takes time to load. Then selenium moves to os_tab to enter root password. 
So we wanted to have all fields should be loaded before moving to os_tab.